lua/tardis/interiors/sequences/legacy_sequences.lua

```diff
@@ -0,0 +1,21 @@
+--Legacy Interior - Control Sequences (advanced mode)
+
+local Seq = {
+    ID = "legacy_sequences",
+
+    ["legacy_longflightdemat"] = {
+        Controls = {
+            "legacy_helmic",
+            "legacy_physbrake",
+            "legacy_handbrake",
+            "legacy_throttle"
+        },
+        OnFinish = function(self)
+            if IsValid(self) and IsValid(self) then
+                self.exterior:Demat()
+            end
+        end
+    }
+}
+
+TARDIS:AddCSequence(Seq)
```

Comment:
'C' for the internal code is cool but for the api surface can we rename this to AddControlSequence?